### DISCUSSION TITLE
Dataset Capstone Data Visualization Specialization

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#701] [Q&A] [answered]
Dataset yang saya gunakan adalah dataset transaksi gojek untuk 4 mode yaitu BIKE, CAR, FOOD, dan SHOP. Saya hanya ingin menggunakan data untuk mode FOOD, tapi setelah di-filter untuk mode FOOD row datanya hanya 265, apakah dengan jumlah row segitu masih bisa digunakan? Ataukah ada saran lain? Untuk row keseluruhannya (semua mode) terdapat 1017.

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @ArdiantheGreat ,

Untuk jumlah data 265 masih dapat digunakan untuk dashboard pada Capstone DV, jika Anda hanya ingin menganalisis untuk mode FOOD dan selama dashboard Bapak masih dapat memenuhi rubrik-rubrik pada Capstone DV maka tetap dianggap relevan dan dapat digunakan. 

Sekian jawaban dari kami, semangat Pak.

**Reply by naturallyfunny** [reply]
Baik terima kasih atas balasannya, sangat membantu.

**Reply by IrfanChairurrachman** [reply]
Sama-sama Pak, kami mark answer untuk jawaban pada diskusi ini ya.

Best Regards,
Team Algoritma
